impact_stack/rest/__init__.py

```diff
@@ -190,16 +177,42 @@ def client_forwarding_as_app(self, request: IncomingRequest, *args, **kwargs) ->
         kwargs["auth"] = self.get_middleware(base_url)
         return self.client(base_url, *args, **kwargs)
 
+
+class BackendClientFactory(ClientFactory):
```

Comment:
Hm, I can see how it's useful to separate clients that need a url pattern from clients that don’t, but I’d argue that both are in the backend, both make app to app requests. What would you think of `PatternClientFactory` or `OwnerClientFactory`?